src/__acceptance_tests__/input-fields-acceptance-test.tsx

```diff
@@ -117,6 +117,16 @@ test.each(STORY_TYPES)('CreditCardExpirationField (%s)', async (storyType) => {
 
     await clearAndType(page, field, '12///34/56');
     expect(await getValue(field)).toBe('12/34');
+
+    await clearAndType(page, field, '222');
+    expect(await getValue(field)).toBe('02/22');
+
+    await page.keyboard.press('Backspace');
+    await page.keyboard.press('Backspace');
+    expect(await getValue(field)).toBe('02');
+
+    await field.type('22');
+    expect(await getValue(field)).toBe('02/22');
```

Comment:
is this the case we try to fix?

trying this in master, the result is almos the same